lib/konbata.rb

```diff
@@ -70,6 +70,7 @@ def self.generate_course_structures(entries)
     end
 
     course_structures.each do |_course_code, volumes|
+      course_structures[_course_code] = volumes.sort.to_h
```

Comment:
Now that `_course_code` is being used, the leading underscore should be removed and it should be `course_code`.